app/Adapters/EnergeticNeeds.php

```diff
@@ -57,18 +57,30 @@ private function calculateBasalEnergyExpenditure(string $gender, float $weight,
         );
     }
 
-    private function calculateTotalEnergyExpenditure(float $basalEnergyExpenditure, string $activity): float
+    public function calculateTotalEnergyExpenditure(float $basalEnergyExpenditure, string $activity): float
     {
+        $avaliableActivities = ['sedentary', 'littleActive', 'active', 'veryActive'];
+
+        $elementExists = in_array($activity, $avaliableActivities, true);
+        if (!$elementExists || $basalEnergyExpenditure < 400) {
```

Comment:
Como a validação aqui é em caso negativo, acredito que poderia torná-la mais semântica atribuindo o valor a variável já negado e deixando o `if` mais espressivo:
```suggestion
        $elementNotExists = !in_array($activity, $avaliableActivities, true);
        if ($elementNotExists || $basalEnergyExpenditure < 400) {
```